queries.yaml

```diff
@@ -273,3 +273,28 @@ post_process:
     params:
       source_layer: roads
       start_zoom: 9
+  - fn: TileStache.Goodies.VecTiles.transform.drop_properties
+    params:
+      source_layer: roads
+      start_zoom: 0
+      end_zoom: 14
+      properties: [is_bridge, is_tunnel, oneway]
```

Comment:
Yes, the sort key is calculated before this. Although that might defeat the purpose... given that the different sort keys will prevent them merging. So perhaps we need to be running this before the sort order calculation (or modify the sort order calculation)?